### ISSUE TITLE
MSM - mismatch

### ISSUE THREAD

**Issue by juansanchezpena** [nipreps/fmriprep#3087]
### What happened?

I was able to run the MSMSulc patch on  GE premiere multiband data at TR = 100 displayed on generic HCP surface. 
The MSM outputs are compared in gif files:

msm = latest msm container with good voxels arg
good voxels = 23.1.4 with good voxels flag
the order int he gif is the order they appear.

msm-abcd
https://imgflip.com/gif/7yvqa6

msm-goodxoxels
https://imgflip.com/gif/7yvqzt

goodvoxels-msm
https://imgflip.com/gif/7yvr3s

The MSM patch seems to have worsened the sulcal distortion

### What command did you use?

```shell
docker service create --constraint 'node.hostname==plisskenmri.nyspi.local' --replicas 1 --reserve-cpu 8 --reserve-memory 32g --mode replicated --restart-condition none --mount type=bind,source=$indir,destination=/input,readonly=true --mount type=bind,source=$outdir,destination=/output  --mount type=bind,source=/MRI_DATA/bin/freesurferlicense.txt,destination=/opt/freesurfer/license.txt,readonly=true   nipreps/fmriprep:msmsulc  /input /output  participant $addarg

-a "--participant-label patensasc0089 --output-spaces T1w MNI152NLin2009cAsym:res-2 --cifti-output --output-layout legacy --project-goodvoxels"
```

### What version of fMRIPrep are you running?

MSMSulc

### How are you running fMRIPrep?

Docker

### Is your data BIDS valid?

Yes

### Are you reusing any previously computed results?

No

### Please copy and paste any relevant log output.

```shell
no errors reported
```

### Additional information / screenshots

msm-abcd
https://imgflip.com/gif/7yvqa6

msm-goodxoxels
https://imgflip.com/gif/7yvqzt

goodvoxels-msm
https://imgflip.com/gif/7yvr3s

**Comment by effigies** [comment]
I suspect we've hit the limits of what we're able to do with just a visual diff of final outputs to go on. To do more, I think we would need to have the working directories of both pipelines that @mgxd and I are both able to access and compare, and probably some time with someone with significant experience to curate them to help track down where results diverge.

**Comment by effigies** [comment]
@Shotgunosine This may line up with what you saw.

**Comment by effigies** [comment]
Fixed in 23.2.0a1.
